src/docs/sdk/research/performance/index.mdx

```diff
@@ -192,6 +192,160 @@ In the example above, if an error bubbles up the call stack we want to be able t
+The requirement of a transaction is specially challenging on frontends (browser, mobile, and desktop applications), because in those cases auto-instrumented transactions are less reliable as they typically only last for some time before being automatically finished.
+
+In our [example trace](#span-ingestion-model), the first span that originates the trace is due to a button click. If the button click `F*` was instrumented as a regular `span`, most likely no data from the frontend would be captured, but the other `B` and `S` spans would still be captured.
+
+In Sentry's model, if a span is not a transaction and there is no ancestor span that is a transaction, then the span won't be ingested. This, in turn, means there are many situations where a trace is missing crucial information that can help debug issues, particularly on the frontend where transactions need to end at some point but execution might continue.
+
+Automatic and manual instrumentation have a challenge deciding whether to start a span or a transaction, and the decision is particularly difficult considering that poor [scope propagation](#scope-propagation) limits the existence of concurrent transactions.
+
+### Missing Web Vital Measurements
+
+Sentry's browser instrumentation collects Web Vital measurements. But, because those measurements are sent along to Sentry using the automatically instrumented transaction as a carrier, measurements that are made available by the browser after the automatic transaction has finished are lost.
+
+This causes transactions to be missing some Web Vitals or to have non-final measurements, for example for the LCP measurement.
+
+### Unreliable Frontend Transaction Duration
+
+Because all data must go in a transaction. Sentry's browser SDK creates a transaction for every page load and every navigation. Those transactions must end at some time.
+
+If the browser tab is closed before the transaction is finished and sent to Sentry, all collected data is lost. Therefore, the SDK needs to balance the risk of losing all data and capturing as much useful data as possible.
+
+Transactions are finished after a certain idle time after the last activity has been observed (for example, outgoing HTTP requests). This means that the duration of the page load / navigation transaction is a rather arbitrary value that can't necessarily be compared or improved, as it doesn't indicate any concrete and undestandable duration.
+
+We counter the limitation by focusing on the LCP Web Vital as the default performance metric for browser, but, as discussed above, the final LCP value might be lost.
+
+### In-memory Buffering Affects Servers
+
+Any system that may operate with several concurrent transactions at one time, which is the case of web servers, will be required to buffer complete span trees in memory.
+
+This means recording 100% of spans and 100% of transactions for many server-side applications, even in a simplified form, is not feasible due to the overhead incurred.
```

Comment:
Rewritten

```suggestion
This means that recording 100% of spans and 100% of transactions is not feasible for many server-side applications, because the overhead incurred is just too high.
```